subscription-manager.spec

```diff
@@ -853,6 +856,14 @@ find %{buildroot} -name \*.py -exec touch -r %{SOURCE0} '{}' \;
 %{_datadir}/omf/subscription-manager/subscription-manager-C.omf
 
 %{_datadir}/applications/subscription-manager-gui.desktop
+%{_datadir}/icons/hicolor/16x16/apps/*.png
+%{_datadir}/icons/hicolor/22x22/apps/*.png
+%{_datadir}/icons/hicolor/24x24/apps/*.png
+%{_datadir}/icons/hicolor/32x32/apps/*.png
+%{_datadir}/icons/hicolor/48x48/apps/*.png
+%{_datadir}/icons/hicolor/96x96/apps/*.png
+%{_datadir}/icons/hicolor/256x256/apps/*.png
+%{_datadir}/icons/hicolor/scalable/apps/*.svg
```

Comment:
I think the listings for icon files are extraneous based on our tests. Suspect this came in via a bad rebase. Going to `commit --amend` and check results.